source/mongodb/data-access.txt

```diff
@@ -0,0 +1,124 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in MongoDB Atlas from client applications using standard
+MongoDB query language syntax with **MongoDB Data Access** in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries
+while your data remains secure on the server.
+
+Realm SDK Support
+-----------------
+
+The following Realm SDKs support MongoDB Data Access:
+
+- :ref:`Java SDK <java-intro>`
+- :ref:`.NET SDK <dotnet-intro>`
+- :ref:`Node.js SDK <node-intro>`
+- :ref:`React Native SDK <react-native-intro>`
+- :ref:`Swift SDK <ios-intro>`
+- :ref:`Web SDK <web-intro>`
+
+MongoDB Data Access **is not** supported in these Realm SDKs:
+
+- C++ SDK
+- Flutter SDK
+- Kotlin SDK
+
+When to Use MongoDB Data Access
+-------------------------------
+
+Some general scenarios when you might want to use MongoDB Data Access in an application:
+
+- You want to use an API like the MongoDB drivers
+- You want to perform aggregations
+- You want to dynamically construct queries from the client device.
+
+You might want to use MongoDB Data Access **instead of** :ref:`Atlas Device Sync <sync>`
+in an application in these scenarios:
+
+- You are querying documents that are not or cannot be modeled in Realm Database.
+- A non-Realm service generates collections that you want to access.
+- You do not want to persist data locally.
+- You are using the Realm Web SDK, which does not support Realm Database or Device Sync.
+
+Limited Server-Side Use
+~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB Data Access is intended for client-side use.
+You should not use the Realm SDKs' Data Access to support significant server-side workloads.
+
+In server-side environments, use the :ref:`Atlas Data API <data-api>` or a MongoDB Driver.
+If you would like to apply App Services Rules to Driver operations,
+you can use the :ref:`App Services Wire Protocol <wire-protocol>`.
+
+.. _client-data-access-set-up:
+
+Set Up MongoDB Data Access
+--------------------------
+
+To use MongoDB Data Access from a Realm SDK, you must perform the following set up.
+
+.. procedure::
+
+   .. step:: Create App Services App
+
+      Create an App Services App. To learn more about App creation,
+      refer to :ref:`Create an App <create-a-realm-app>`.
```

Comment:
We should have a step (or at least a "Make sure to...") for **Link a Data Source**

Reply:
absolutely. worth the call out.